db/seeds/source_types.yml

```diff
@@ -264,6 +264,20 @@ openshift:
           :is: true
   :vendor: Red Hat
   :icon_url: "/apps/frontend-assets/platform-logos/openshift-container-platform.svg"
+rh-marketplace:
+  :product_name: Red Hat Marketplace
+  :vendor: Red Hat
+  :schema:
+    :authentication:
+      - :type: token
+        :name: API Token
+        :fields:
+          - :component: text-field
+            :name: application.extra.api_token
+            :label: API Token
+            :isRequired: true
+            :validate:
+              - :required:
```

Comment:
Not sure how this yaml is being translated to json, but I think it should look like this: `        - :type: required `

the result should like this in JSON format:
`validate: [{ type: “required” }]`

Reply:
If you are interested, here is a documentation for the schema we are using here: https://data-driven-forms.org/schema/introduction

Reply:
There is also a sources wizard guide: https://github.com/RedHatInsights/sources-ui/blob/master/doc/update-wizard.md

Reply:
Fixed. Thank you for the links!